modules/eslint-plugin/spec/rules/no-dispatch-in-effects.spec.ts

```diff
@@ -51,6 +51,42 @@ class Ok2 {
       { dispatch: false },
     )
   }
+}`,
+  `
+import { Store } from '@ngrx/store'
+import { inject } from '@angular/core'
+
+class Ok3 {
+  private readonly actions = inject(Actions);
```

Comment:
Because this is a test using `Actions`, I think it's worth adding an additional test where`inject` is imported from somewhere else than `@angular/core` .

Just to make sure things work as expected and don't throw.